src/api/db/job.ts

```diff
@@ -26,21 +14,24 @@ export function getJob(jobId: number): Promise<any> {
       .then(job => {
         getBuild(job.builds_id)
```

Comment:
result of that additional query is not necessary included in result from previous query. In that additional query we are looking last successfull job_run with same atttributes. This run can belong to other build... Maybe I should rename variable, to be more clear what this is...